ci/steps/brew-bump.sh

```diff
@@ -5,6 +5,21 @@ main() {
   cd "$(dirname "$0")/../.."
   # Only sourcing this so we get access to $VERSION
   source ./ci/lib.sh
+  source ./ci/steps/steps-lib.sh
+
+  echo "Checking environment variables"
+
+  # We need VERSION to bump the brew formula
+  if [[ $(is_env_var_set "VERSION") -eq 1 ]]; then
```

Comment:
If your function returns 0, it's considered true (zero exit code means success) and if it returns 1 then it's considered false. That means you can also use a pattern like this, which looks a little nicer:

```bash
#!/bin/bash

set -euo pipefail

is_true() {
  return 0
}

is_false() {
  return 1
}

if is_true; then
  echo "is_true: true"
else
  echo "is_true: false"
fi

if is_false; then
  echo "is_false: true"
else
  echo "is_false: false"
fi
```

This program prints:

```shell-session
$ bash test.sh
is_true: true
is_false: false
```

Reply:
Oh I like this way better. I'll refactor to do this. Thank you!